eliot-service/eliot/symbolicate_resource.py

```diff
@@ -149,6 +155,58 @@ def validate_stacks(stacks, modules):
                 )
 
 
+def bytes_split_generator(item, sep):
```

Comment:
SYM files are _LARGE_, so I needed something that would parse just the part I need to look at.